pyramid_oereb/lib/records/plr.py

```diff
@@ -172,14 +161,21 @@ def calculate(self, real_estate):
             if geometry.calculate(
                     real_estate,
                     self.min_length, self.min_area,
-                    self.length_unit, self.area_unit):
+                    self.length_unit, self.area_unit
+            ):
                 tested_geometries.append(geometry)
                 inside = True
         self.geometries = tested_geometries
         length = self._sum_length()
-        self._length = None if length is None else round(length, self.length_precision)
+        if length is None:
+            self._length = None
+        else:
+            self._length = int(round(length, 0))
         area = self._sum_area()
-        self._area = None if area is None else round(area, self.area_precision)
-        self.part_in_percent = None if area is None else \
-            round(((area / real_estate.limit.area) * 100), self.percentage_precision)
+        if area is None:
+            self._area = None
+            self.part_in_percent = None
+        else:
+            self._area = int(round(area, 0))
+            self.part_in_percent = round(((area / real_estate.land_registry_area) * 100), 1)
```

Comment:
no `int()` ?

Reply:
No, the specification defines "PartInPercent" as decimal number with one decimal place.
https://github.com/camptocamp/pyramid_oereb/issues/458#issuecomment-365928946